test/fail_compilation/test18554.d

```diff
@@ -1,24 +0,0 @@
-/* REQUIRED_ARGS: -preview=dip1000
-EXTRA_FILES: imports/imp18554.d
```

Comment:
Why is this test removed instead of changing it to match the new/old behavior?

Reply:
The tests were for code that isn't there anymore. I didn't think it made sense to test the absence of a special case, it's not something you would accidentally write again.

Reply:
True. But a quick search didn't yield any other test for visibility checks w.r.t. `tupleof` - might be good to either adapt it or integrate it into another test